src/main/kotlin/com/cognifide/gradle/aem/tooling/clean/Cleaner.kt

```diff
@@ -171,8 +168,6 @@ class Cleaner(project: Project) {
     private fun deleteFiles(root: File) {
         if (root.isDirectory) {
             aem.project.fileTree(root, filesDeleted).forEach { deleteFile(it) }
-        } else {
-            deleteFile(root)
```

Comment:
why?

Reply:
for instance if root is a file, should be remained, not removed